integrators.md

```diff
@@ -17,22 +19,25 @@ This process is intended to be followed for large technical initiatives and not
 3. Identify all the ShapeShift owned repositories that your feature will need to modify or extend, please read all documentation in those associated repositories.
-4. Obtain an approved design document. Integrating teams should create a design document for their planned feature. Previous examples of these documents can be found [here](designs.md). Your document should start out as a Google document that can be used for questions and comments. Once you believe the document is completed, please reach out to @joshf on Discord to schedule a design review meeting. The first half of the meeting will be spent with core engineers individually reading your design document (in silence). The second half of the meeting will be a discussion of questions, comments or any points that need additional clarification or changes. A determination will be made from the core workstream of either approval to move forward or clear next steps to obtain that approval. Once your document is approved, please open a PR to add it to our [docs repo](https://github.com/shapeshift/docs).
+4. Obtain approval from Product Workstream on Feature Spec and UI/UX mockups. Depending on bandwidth, the Product Workstream may be able to provide both of these upon request. Alternatively, you can submit draft specs and mockups to the workstream for feedback and approval and can request assistance at any time in the #product-design-public channel in the [ShapeShift Discord](https://discord.gg/shapeshift). To access the channel, select the Product role in the #role-selector channel.
+    1. [Feature Spec Template & Guide](https://www.notion.so/shapeshift/Product-Feature-Spec-Template-17acd69df70e446fa5656fdaaeccf966) & [Examples](https://www.notion.so/shapeshift/Product-and-Creative-Services-Workstream-16fcd5de93334a389adc656a8bedb570)
+    2. [UI/UX Components & Mockup Templates](https://www.figma.com/file/OHbVrAVaV2xQWUsq0nKzGm/ShapeShift-Library?node-id=25%3A576) & [Example](https://invis.io/WF1262ZUU2Q6)
+6. Obtain an approved design document. Integrating teams should create a design document for their planned feature. Previous examples of these documents can be found [here](designs.md). Your document should start out as a Google document that can be used for questions and comments. Once you believe the document is completed, please reach out to @joshf on Discord to schedule a design review meeting. The first half of the meeting will be spent with core engineers individually reading your design document (in silence). The second half of the meeting will be a discussion of questions, comments or any points that need additional clarification or changes. A determination will be made from the core workstream of either approval to move forward or clear next steps to obtain that approval. Once your document is approved, please open a PR to add it to our [docs repo](https://github.com/shapeshift/docs).
```

Comment:
Nitpick: `Google Document`, as we are referring to the Google product, Document.

Reply:
Also, is this still @joshf, or is it now @0xdef1cafe?

If it was still Josh, it's now @joshaf 😉

Reply:
fixed both, ty!